en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,52 @@
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
 
 ## API Management Components
-Described below are the key components of Choreo API management:
+API management in Choreo is facilitated by the key components: [API Publisher](#api-publisher), [Developer Portal](#developer-portal), [API Key Manager](#api-key-manager), [Choreo Connect](#choreo-connect), and [Traffic Manager](#traffic-manager).  
 
 ### API Publisher 
-The API Publisher is designed for API creators to develop, document, secure, test, and version APIs with ease.
+The API Publisher is designed for API creators to develop, document, secure, test, and version APIs with ease. It also caters to more API management-related tasks, such as allowing API Publishers to deploy and publish APIs and applying rate-limiting policies.
 
 ### Developer Portal
-The Developer portal allows API publishers to host and advertise their APIs while allowing API consumers to self-register, discover, evaluate, subscribe to, and consume APIs securely and easily.
+The Developer portal allows API publishers to host and advertise their APIs and API consumers to discover, evaluate, subscribe to, and consume APIs securely.
 
 ### API Key Manager
-API Key Manager handles all security and key-related operations. 
+The API Key Manager is the identity provider for Choreo and acts as the Secure Token Service (STS). Choreo's API management supports OAuth2.0 and API-Key-based authentication mechanisms.
+
+In Choreo, tokens are bound to an application. The Key manager provides a token API to generate access tokens. Clients can use these tokens to invoke APIs exposed by Choreo. The Key Manager also exposes a revoke token API that clients can use to revoke an access token. A client can generate an OAuth2.0 access token by invoking the token API directly or via the Developer Portal UI. Alternatively, a client can generate an API Key through the Developer Portal without calling the Key Manager. The API Key is a self-signed JWT token. When a client invokes an API with an OAuth2.0 access token or an API Key, the Gateway validates the token by validating its signature and subscription.
+
+The Key Manager performs scope validation as well.
+
+### Choreo Connect
+
+Choreo Connect is an API gateway designed for microservices that is cloud-native, decentralized, and developer-centric. This API gateway is a lightweight message processor for APIs that facilitates message security, transport security, routing, and other API Management functions such as throttling, rate-limiting, etc.
+
+Choreo Connect consists of three components: Router, Enforcer, and Adapter.
+
+![Choreo Connect](../assets/img/apis/choreo-connect.png){:style="height:75%;width:90%"}
```

Comment:
Shouldn't we re-draw this with choreo colors? Otherwise having this in orange won't suit for choreo docs.

Reply:
Holding this off, until we figure out the colors we want to use for Choreo